resources/public/api/conf/2.0/application.yaml

```diff
@@ -70,6 +70,20 @@ paths:
               example:
                 code: SCHEMA_VALIDATION
                 message: Request failed schema validation
+        '406':
+          description: Not Acceptable
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: '#/components/schemas/AcceptHeaderInvalid'
+                  - description: Accept header application/vnd.hmrc.2.0+json123 is not supported!.
+                    example:
+                      code: NOT_ACCEPTABLE
+                      message: Accept header application/vnd.hmrc.2.0+json123 is not supported!
+              example:
+                code: NOT_ACCEPTABLE
+                message: Accept header application/vnd.hmrc.2.0+json123 is not supported!

```

Comment:
A few issues:

- Up to now, the preferred `description` for a `406` was 'The Accept header is missing or invalid.'
- Why write` json123`?
- Why use an exclamation mark (`!`)?

Reply:
To be fair, that was already in the code like that from before. I imagine this was masked because API Platform overrides the messages for many of the codes.

I don't disagree with what you said though, we should fix the messages in our code to fit the style of all other messages, platform provided or otherwise.

Reply:
"The Accept header is missing or invalid."